qa-heal.planx-pla.net/manifests/hatchery/hatchery.json

```diff
@@ -2,12 +2,8 @@
   "user-namespace": "jupyter-pods-qa-heal",
   "sub-dir": "/lw-workspace",
   "user-volume-size": "10Gi",
-  "gen3-license-user-maps-dynamodb-table": "gen3-license-user-maps",
-  "gen3-license-type": "STATA-HEAL",
-  "gen3-license-max-ids": 6,
-  "gen3-g3auto-name": "stata-workspace-gen3-license-g3auto",
-  "gen3-g3auto-key": "stata_license.txt",
-  "gen3-license-file-path": "stata.lic",
+  "license-user-maps-dynamodb-table": "gen3-license-user-maps",
+  "license-user-maps-global-secondary-index": "activeUsersIndex",
```

Comment:
just wonder why this is called `secondary index`?

Reply:
The table already has a [primary key](https://github.com/uc-cdis/cloud-automation/blob/355cbfc0676b807cae0c2d7d9acd0b38df1291af/gen3/bin/kube-setup-hatchery.sh#L51-52). This key has the intent that table items are distributed evenly across partitions. The primary key can be used for a scan on all items in the table. The global secondary index is created for the use case of a query (not a scan) for active license-user sessions.